engine/namegen.ml

```diff
@@ -218,21 +218,21 @@ let it_mkLambda_or_LetIn_name env sigma b hyps =
 let get_mangle_names =
   Goptions.declare_bool_option_and_ref
     ~stage:Summary.Stage.Interp
-    ~depr:false
+    ~depr:None
```

Comment:
How about making `~depr:None` the default rather than repeating that in so many places?

Reply:
Doesn't work because all arguments are labelled

Reply:
Changed to add a unit argument and make stage and depr optional